src/reactviews/pages/SchemaCompare/SchemaCompare.tsx

```diff
@@ -66,9 +66,7 @@ export const SchemaComparePage = () => {
 
             {!showMessage() && <SchemaDifferences onDiffSelected={handleDiffSelected} />}
```

Comment:
I'd expect that the current diff selection also gets piped through to the list of schema differences so that row can be highlighted.